__tests__/games.test.js

```diff
@@ -3,6 +3,7 @@ const app = require("../app.js")
 const db = require("../db/connection.js")
 const seed = require("../db/seeds/seed")
 const testData = require("../db/data/test-data/index")
+require('jest-sorted');
```

Comment:
This shouldnt be here, re read the docs. We should be adding to the package.json instead